packages/contracts/src/DefiLlamaOracle.sol

```diff
@@ -48,8 +50,24 @@ contract DefiLlamaOracle is TrustedOracle {
     ///       as expected.
     ///     - `uint256 measurementTimestamp`: a timestamp that indicates when the final measurement
     ///       should be taken and posted on-chain.
-    function _initialize(InitializeOracleParams memory _params) internal override {
-        (string memory _specification, uint256 _measurementTimestamp) = abi.decode(_params.data, (string, uint256));
+    ///     - `Constraint constraint`: since this is a constrained oracle, this is used to specify
+    ///       the constraint type.
+    ///     - `uint _value0`: since this is a constrained oracle, this is used to specify the first
+    ///       value of the constraint.
+    ///     - `uint _value1`: since this is a constrained oracle, this is used to specify the second
+    ///       value of the constraint.
+    function initialize(InitializeOracleParams memory _params) external payable override initializer {
```

Comment:
Why did the `initialize` become `payable`?

Reply:
Previously, the `initialize` function defined here was an internal function coming from the extended `TrustedOracle` contract. This instead is the implementation of the `initialize` function that's defined in the `IOracle` core interface, which is payable. The difference is that previously the payable initialize function was defined in the TrustedOracle, which then called this internal `initialize` which didn't require the payable specifier.